samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -15,7 +17,8 @@ namespace MicrosoftEduGraphSamples.Workflows
     internal class AssignmentWorkflow
     {
         private readonly IConfiguration _config;
-
+        private const int MAX_RETRIES = 10; // Maximum number of retries for long running operations
```

Comment:
is this constant still used here?, remove it if not

Reply:
removed